core/services/workflows/metering/metering.go

```diff
@@ -145,11 +151,26 @@ func (r *Report) Reserve(ctx context.Context) error {
 
 	// If there is no credit limit defined in the workflow, then open an empty reservation
 	// TODO: https://smartcontract-it.atlassian.net/browse/CRE-284 consume user defined workflow execution limit
+	chainSelector, err := strconv.ParseUint(r.workflowRegistryChainSelector, 10, 64)
```

Comment:
To make this more DRY, maybe either do this in a utili, or once in the constructor